synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -1064,6 +1080,9 @@ Dialog_Setup::refresh()
 	// Refresh the sequence separator
 	image_sequence_separator.set_text(App::sequence_separator);
 
+	// Refresh the number of core
+	number_of_core_select->set_value(int(adj_number_of_core->get_value()));
```

Comment:
This isn't right: `refresh()` is to update the form fields, and not set the field with its "own" value.
You could keep the App "member" variable you created commits ago, but now removed and used it here :) Something like:
`number_of_core_select->set_value(App::num_threads);`

Reply:
@rodolforg I think threads is better. We can't change CPU cores :)